src/http/api/views/account.follows.view.ts

```diff
@@ -179,11 +226,90 @@ export class AccountFollowsView {
             return error('error-getting-follows');
         }
 
-        if (!page) {
-            return error('no-page-found');
-        }
+        // Handling non-paginated follow lists
+        if (!page || Number(next)) {
+            const follows =
+                type === 'following'
+                    ? await actor.getFollowing()
+                    : await actor.getFollowers();
 
-        const accounts: AccountInfo[] = [];
+            if (!follows) {
+                return error('error-getting-follows');
+            }
+
+            console.log('Follows: ', follows);
+
+            const pageSize = 15;
+            const pageNumber = next ? Number.parseInt(next, 10) : 1;
+            const startIndex = (pageNumber - 1) * pageSize;
+
+            const pageUrls = follows.itemIds.slice(
+                startIndex,
+                startIndex + pageSize,
+            );
+
+            for await (const item of pageUrls) {
+                try {
+                    const followsActorObj = await lookupObject(item.href, {
+                        documentLoader,
+                    });
+
+                    if (!isActor(followsActorObj)) {
+                        continue;
+                    }
+
+                    const followeeAccount = await this.db('accounts')
```

Comment:
If we're going to check our local db - let's do that before calling `lookupObject` and just use our local data? That way we save a network lookup!